### PR TITLE
(2225) Add the regulation type filter to the public pages

### PR DESCRIPTION
**Pull request by bryangaledxw**
# Changes in this PR

Support for searching by regulation type in public-facing Profession and Organisation search

### Before
localhost_3000_professions_search
localhost_3000_regulatory-authorities_search

### After
localhost_3000_professions_search_keywords= regulationTypes%5B%5D=licensing
localhost_3000_regulatory-authorities_search_keywords= regulationTypes%5B%5D=licensing regulationTypes%5B%5D=accreditation

### CODE REVIEW

File: cypress/integration/professions/search/search.spec.ts [hunk 1]
```diff
@@ -52,12 +52,37 @@ describe('Searching a profession', () => {
...
+    cy.get('input[name="keywords"]').should('have.value', 'Attorney');
+
+    cy.get('body').should('contain', 'Registered Trademark Attorney');
+    cy.get('body').should(
+      'not.contain',
+      'Secondary School Teacher in State maintained schools (England)',
+    );
+  });
+
+  it('The search uses a professions keywords to filter', () => {
```
Comment (Gweaton) [inline cypress/integration/professions/search/search.spec.ts:70]: Is this a duplicate of the test above?

**Review by Gweaton — APPROVED** [review]

File: cypress/integration/professions/search/search.spec.ts (hunk 1 above)
Comment (bryangaledxw) [reply cypress/integration/professions/search/search.spec.ts:70]: Ah, no, this was just me copying and paste-ing a test from below so we're keeping to a consistent order!

And the difference in these two tests is one is searching on "Attorney" (matches the title) and one is searching "Adjudicator" (matches the keywords)

Comment (bryangaledxw) [reply cypress/integration/professions/search/search.spec.ts:70]: I've pulled the re-ordering into its own commit, makes things a lot clearer!

Comment (Gweaton) [reply cypress/integration/professions/search/search.spec.ts:70]: ahh I see, thanks for clarifying

**Pull request merged**